source/helpers/sass_helpers.ts

```diff
@@ -0,0 +1,16 @@
+function getImplStatus(status) {
```

Comment:
@jgerigmeyer would love some help figuring out how to use this helper method in a new Eleventy filter with types for the `status` arg. I had something like this: `function getImplStatus(status: string | boolean | null){` but then the build command breaks with this error: 
```
[11ty] 1. Error in your Eleventy config file 'eleventy.config.cjs'. (via EleventyConfigError)
[11ty] 2. Unexpected token ':' (via SyntaxError)
[11ty] 
[11ty] Original error stack trace: /Users/sanajaved/Documents/oddbird/sass-site/source/helpers/sass_helpers.ts:1
[11ty] function getImplStatus(status: string | boolean | null){
```
The tsconfig looks like it should include this file since its inside source so I'm not sure what else to try and tinker with 🤔

Reply:
Should be fixed in https://github.com/oddbird/sass-site/pull/21